.github/workflows/ci.yml

```diff
@@ -109,13 +114,15 @@ jobs:
                 apt-get -o Acquire::Retries=$NET_RETRY_COUNT update
                 apt-get -o Acquire::Retries=$NET_RETRY_COUNT install -y g++ python libpython-dev git
             fi
-            if [[ "${{ matrix.coverity }}" == "yes" ]]; then
+            # multiple job types are not compatible with ccache, they use "ccache: no" in the matrix
+            if [[ "${{ matrix.ccache }}" == "no" ]]; then
                 echo "B2_USE_CCACHE=0" >> $GITHUB_ENV
             fi
             git config --global pack.threads 0
 
       - uses: actions/checkout@v2
         if: '!matrix.coverage'
+
```

Comment:
I purposely did not add empty lines between steps, that should be 1 if that was possible, so they are at least visually related. Here and https://github.com/boostorg/boost-ci/pull/130/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR144
But well... At least this step can maybe be combined: #133 tests that.